pkgs/top-level/all-packages.nix

```diff
@@ -25783,9 +25783,9 @@ in
   spdlog = spdlog_1;
 
   dart = callPackage ../development/interpreters/dart { };
-  dart_stable = dart.override { version = "2.0.0"; };
-  dart_old = dart.override { version = "1.24.3"; };
-  dart_dev = dart.override { version = "2.0.0-dev.26.0"; };
+  dart_old = dart.override    { version = "1.24.3";         };
+  dart_stable = dart.override { version = "2.7.1";          };
+  dart_dev = dart.override    { version = "2.8.0-dev.10.0"; };
```

Comment:
I know this looks nice, but it's better if editors just vertically aligned in the buffer.
Could you revert it? It's not a codestyle we have.

Reply:
ah okay - sure thing